From 4cf44e7c166820172063276839edb4636c20587a Mon Sep 17 00:00:00 2001
From: Madalin Bucur <madalin.bucur@freescale.com>
Date: Fri, 7 Feb 2014 11:45:46 +0200
Subject: [PATCH 096/466] dpaa_eth: minor Makefile changes

Minor Makefile changes.

Signed-off-by: Madalin Bucur <madalin.bucur@freescale.com>
Change-Id: Ibec913fdae43bf785852e803e1821d1f651272f5
Reviewed-on: http://git.am.freescale.net:8181/8645
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Cristian-Constantin Sovaiala <Cristian.Sovaiala@freescale.com>
Reviewed-by: Ruxandra Ioana Radulescu <ruxandra.radulescu@freescale.com>
Reviewed-by: Marian Cristian Rotariu <marian.rotariu@freescale.com>
[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]
Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 drivers/net/ethernet/freescale/dpa/Makefile |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/net/ethernet/freescale/dpa/Makefile b/drivers/net/ethernet/freescale/dpa/Makefile
index e0196f7..13d4b06 100644
--- a/drivers/net/ethernet/freescale/dpa/Makefile
+++ b/drivers/net/ethernet/freescale/dpa/Makefile
@@ -3,7 +3,7 @@
 #
 ccflags-y += -DVERSION=\"\"
 #
-#Include netcomm SW specific definitions
+# Include netcomm SW specific definitions
 include $(srctree)/drivers/net/ethernet/freescale/fman/ncsw_config.mk
 
 ccflags-y += -I$(NET_DPA)
@@ -12,7 +12,7 @@ obj-$(CONFIG_FSL_DPAA_1588) += dpaa_1588.o
 # dpaa_debugfs needs to be initialized before dpaa_eth
 obj-$(CONFIG_FSL_DPAA_ETH_DEBUGFS) += dpaa_debugfs.o
 obj-$(CONFIG_FSL_DPAA_ETH) += fsl-mac.o fsl-dpa.o fsl-dpa-common.o \
- fsl-dpa-base.o fsl-dpa-shared.o fsl-dpa-macless.o fsl-dpa-proxy.o
+ fsl-dpa-base.o fsl-dpa-shared.o fsl-dpa-proxy.o fsl-dpa-macless.o
 obj-$(CONFIG_FSL_DPAA_OFFLINE_PORTS) += fsl-oh.o
 
 fsl-dpa-objs	:= dpaa_ethtool.o dpaa_eth_sysfs.o \
@@ -20,10 +20,10 @@ fsl-dpa-objs	:= dpaa_ethtool.o dpaa_eth_sysfs.o \
 fsl-dpa-common-objs	:= dpaa_eth_common.o
 fsl-dpa-base-objs	:= dpaa_eth_base.o
 fsl-dpa-shared-objs	:= dpaa_eth_shared.o
-fsl-dpa-macless-objs	:= dpaa_eth_macless.o
 fsl-dpa-proxy-objs	:= dpaa_eth_proxy.o
+fsl-dpa-macless-objs	:= dpaa_eth_macless.o
 fsl-mac-objs	:= mac.o mac-api.o
 fsl-oh-objs	:= offline_port.o
 
-#Needed by the tracing framework
+# Needed by the tracing framework
 CFLAGS_dpaa_eth.o := -I$(src)
-- 
1.7.10.4

